cmd/client/main.go

```diff
@@ -0,0 +1,154 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func main() {
+	for {
```

Comment:
[CHANGE REQUEST] Please review this code. ChatGPT made several errors.
Anyway, it should be in the example package.

Reply:
Just to clarify, if I've modified the original client from the course to fit my own style, does that mean I'm actually using it?
Also, I don't understand why it would be moved to the example package, since it's a functional part of the client. I took the basic structure from the course

Reply:
If it's not an example, you need to follow the best practice and guidelines.

https://github.com/uber-go/guide/blob/master/style.md